src/python/setup.py

```diff
@@ -163,6 +173,7 @@ def do_not_install(require):
         "pillow >= 5.2.0",
         "prettytable == 0.7.2",
         "resampy == 0.2.1",
+        "llvmlite == 0.31.0",
```

Comment:
I don't think you want this change. Hoyt made a better change for this which is already in master (i.e. only pin llvmlite version for affected versions of Python).

Reply:
Yeah, I can remove this.